src/internals.cc

```diff
@@ -22,10 +22,37 @@
 
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
+#include <gul14/join_split.h>
+#include <gul14/SmallVector.h>
+
 #include "internals.h"
 
 namespace task {
 
 const gul14::string_view abort_marker{ u8"\U0001F6D1ABORT\U0001F6D1" };
 
+ErrorCause remove_abort_markers_from_error_message(std::string& msg)
```

Comment:
Looks ok 👍 ! But I don't like the wired things about splitting a string on `abort_marker` and examine its result.

Reply:
I'll answer that together with Fini's "out-of-band" question below.